NTupleAnalyzerHH_nanoAOD.py

```diff
@@ -893,37 +973,37 @@ def LeptonsAndJetsFromHH(T):
 	recoelectroninds = []
 	onShellZMu=False
 	onShellZEle=False
+	matchedrecomuons=[]
+	matchedrecoelectrons=[]
+	emptyvector = TLorentzVector()
+	emptyvector.SetPtEtaPhiM(0,0,0,0)
 
+###################### genMatching Muons ##########################
 	for n in range(len(T.Muon_pt)):
 		m = TLorentzVector()
-		m.SetPtEtaPhiM(T.Muon_pt[n],T.Muon_eta[n],T.Muon_phi[n],0)
+		m.SetPtEtaPhiM(T.Muon_pt[n],T.Muon_eta[n],T.Muon_phi[n],T.Muon_mass[n])
```

Comment:
This looks like a problem in the indentation compared to the old version, please check.